pkg/k8s/rule_translate_test.go

```diff
@@ -71,7 +140,7 @@ func (s *K8sSuite) TestTranslator(c *C) {
 	c.Assert(len(rule.ToCIDRSet), Equals, 1)
 	c.Assert(string(rule.ToCIDRSet[0].Cidr), Equals, epIP+"/32")
 
-	translator = NewK8sTranslator(serviceInfo, endpointInfo, true)
+	translator = NewK8sTranslator(serviceInfo, endpointInfo, true, svcLabels)
```

Comment:
Not sure if relevant, but do you want to add checks and tests with svcLabels == nil?

Reply:
Thanks for the catch, but I think that empty svcLabels case is tested in `TestTranslatorDirect`.